src/sync/pull.rs

```diff
@@ -1203,6 +1297,7 @@ mod tests {
                 intervening_sync: false,
                 exp_replay_ids: vec![],
                 exp_err: None,
+                exp_changed_keys: ChangedKeysMap::new(),
```

Comment:
@phritz It seems like this test are all about nop mutations and pulls? I was looking for a rust/wasm test that does pull with changes but I guess we didn't need to test this before?